test/imp/sum-save-proofs-spec.k.save-proofs.kore.golden

```diff
@@ -110,21 +110,17 @@ module haskell-backend-saved-claims-43943e50-f723-47cd-99fd-07104d664c6d
                         /* Fl Fn D Spa */
                         /* InternalMap: */ Lbl'Unds'Map'Unds'{}(
                             /* concrete element: */ Lbl'UndsPipe'-'-GT-Unds'{}(
-                                /* Fl Fn D Sfa Cli */
                                 /* Inj: */ inj{SortId{}, SortKItem{}}(
-                                    /* Fl Fn D Sfa Cl */
-                                    \dv{SortId{}}(/* Fl Fn D Sfa Cl */ "n")
+                                    \dv{SortId{}}("n")
```

Comment:
This is a `Key`, so the `Unparse` instance isn't going through `TermLike` (which is what creates the comments). The comments aren't necessary because all `Key` are functional constructor-like patterns.